classes/04class/exercises/c04-iac02/README.md

```diff
@@ -1,48 +1,17 @@
-# VPC (c04-iac02)
+# C04-IAC02
```

Comment:
this file should not be changed...

Reply:
Hey Kiko!
I did not change the file directly.
I just renamed the ANSWER.md to README.md!